data/map planets.txt

```diff
@@ -1607,6 +1626,19 @@ planet Furnace
 		threshold 3500
 		fleet "Large Syndicate" 7
 
+planet Gagarin
+	attributes ccor factory urban
+	landscape land/city0
+	description `Gagarin is the most populous planet in what used to be the Confederated Councils of the Outer Rim, and it was the first to be settled in this region. All of the settlements on Gagarin are connected to a planetary transport and infrastructure grid... although the grid hasn't been able to keep pace with Gagarin's growing population. Despite having been designed for expansion commensurate with increasing demand, such a resource-intensive infrastructure project became harder and harder to maintain due to continuous supply shortages. The collapse of the CCOR has only exacerbated the grid's deficiencies.`
```

Comment:
```suggestion
	description `Gagarin is the most populous planet in what used to be the Confederated Councils of the Outer Rim, and was one of the first colonies established in this region. All of the settlements on Gagarin are connected to a planetary transport and infrastructure grid... although the grid hasn't been able to keep pace with Gagarin's growing population. Despite having been designed for expansion commensurate with increasing demand, such a resource-intensive infrastructure project became harder and harder to maintain due to continuous supply shortages. The collapse of the CCOR has only exacerbated the grid's deficiencies.`
```
Now that I think about it, Thule could easily be confused with "this region" (the Other Rim), which is why I've changed it to `one of the first colonies established in this region`.

I'd like another opinion here too.

Reply:
The intention with CCOR space is that it was settled around the same time as Thule, part of the wave of colonization spurred on by billionaire 'philanthropists'. Didn't go so well in CCOR space leading the populace to have a revolution that formed CCOR.

That being said since the line wouldn't be clear to the player than Men, two jumps away, isn't part of this small region I think this rephrasing is appropriate.

Reply:
@bene-dictator if you remove `it`, then you need to remove the preceding comma too.

Reply:
~~Don't tell anyone but I don't actually know any grammar rules.~~